### DISCUSSION TITLE
File System level backup/restore

### DISCUSSION THREAD

**Discussion by 2020testuser** [vmware-tanzu/velero#3332] [Community support Q&A] [answered]
**What steps did you take and what happened:**
I want to take a backup of my entire Kubernetes cluster including the entire filesystem (under /etc/kubernetes).  When i created  the cluster backup using the below command, it only takes the cluster resources. 

./velero backup create cluster-backup-1250 --include-cluster-resources

**What did you expect to happen:**

The filesystems used by the cluster must also need to be backed up.

**Environment:**

- Velero version (use `velero version`): 1.5.2
- Velero features (use `velero client config get features`): 
- Kubernetes version (use `kubectl version`):
- Kubernetes installer & version:on1.16.x on-prem
- Cloud provider or hardware configuration: No Cloud Providers
- OS (e.g. from `/etc/os-release`): Linux

**Comment by zubron** [comment, marked as answer]
Hi @2020testuser. Unfortunately, this is not possible with Velero as it is designed to backup Kubernetes resources and persistent volumes and does so via the Kubernetes API. It does not have direct access to the node filesystems. Please see the second question under our [Restic FAQ](https://github.com/vmware-tanzu/velero/wiki/General-FAQ#restic).
